ddtrace/internal/remoteconfig/worker.py

```diff
@@ -147,8 +150,8 @@ def update_product_callback(self, product, callback):
         """
         return self._client.update_product_callback(product, callback)
 
-    def register(self, product, pubsub_instance, skip_enabled=False):
-        # type: (str, PubSub, bool) -> None
+    def register(self, product, pubsub_instance, skip_enabled=False, restart_on_fork=False):
```

Comment:
What happens If they don't register as `restart_on_fork`? they lose the current configurations and have to re-register?

Reply:
Currently any service that registers RC and needs to be restarted after fork is implementing custom logic to re-register/update with the RC client. This change offers a standardised way of doing that so that products can spare the custom logic. An example of this is in #10251.